.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Context
+
+<!-- Why do we need this PR? What was the reason that lead you to make this change? -->
+
+## Objective
+
+<!-- What does this PR fix? What intentional changes will this PR make? -->
+
+## References
+
+<!-- Links or resources that help clarify and support your intentions -->
```

Comment:
The templates only pre-populate the information in the description.

It doesn't enforce it 😇